drivers/platform/nintendo3ds/ctr_sdhc.h

```diff
@@ -0,0 +1,147 @@
+// SPDX-License-Identifier: GPL-2.0
+/*
+ *  Nintendo 3DS Secure Digital Host Controller driver
+ *
+ *  Copyright (C) 2021 Santiago Herrera
+ */
+
+#define SDHC_HCLK	67027964	/* ~67MHz */
```

Comment:
Should this come from the device tree?
And does it come from a configurable PLL or some clockgen device?

Reply:
probably yeah, it's the same freq as the NDS clk (or 1/2 the DSi cpu clk, or 1/4 the 3DS cpu clk)
other devices might also use it, so I'll just add more clocks derivated from the 133MHz clock (*2 and /2)